lib/generators/rolemodel/saas/devise/templates/app/views/devise/registrations/new.html.slim

```diff
@@ -1,15 +1,15 @@
 .flex.flex-col.items-center
-  h2.margin-top-lg Sign up
+  h2.margin-top-lg= t(".sign_up", default: "Sign up")
   .card.container--xs.margin-top-md
     = simple_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f|
       = f.input :first_name, required: true, autofocus: true
       = f.input :last_name, required: true
       = f.simple_fields_for :organization do |organization_field|
-        = organization_field.input :name, label: 'Organization name'
+        = organization_field.input :name, label: t(".organization_name", default: "Organization name")
```

Comment:
Shouldn't the normal AR translations handle this? Do we really need to set the label manually?

Reply:
I'll give it a look. I feel like we need to override it because it was defaulting to just "Name" which isn't very descriptive in this context.

Reply:
So, it does just show "Name" as expected, but I changed it to have organization name be in the devise.en.yml to be translated as "Organization name".